src/Containers/PanelCard/PanelCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useCallback } from 'react';
+import { Button } from '@Inputs/Button/Button';
+import { CardProps, Card } from '../Card/Card';
+
+export enum OperationState {
+    isLoading,
+    isFailure,
+    isSuccess,
+    isUnknown,
+}
+
+export interface IPanelCardProps extends CardProps {
+    messageIsFailure?: string;
+    messageIsSuccess?: string;
+    messageIsLoading?: string;
```

Comment:
common strings are at the end.

isFailureMessage
isSuccessMessage
isLoadingMessage